src/Components/UserList/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './style.module.css';
+import Status from '../Status';
+
+const testData = [
+  {
+    name: "Zach Gallaway",
+    businessName: "Food 4 U",
+    dateRegistered: "2020/07/07",
+    status: "active"
+  },
+  {
+    name: "Maxwell Countryman Skewes",
+    businessName: "Max's Country Meats",
+    dateRegistered: "2018/06/12",
+    status: "pending"
+  },
+  {
+    name: "Ian Scott",
+    businessName: "Scotts Tots",
+    dateRegistered: "2014/01/11",
+    status: "inactive"
+  },
+  {
+    name: "Jerry Goodman",
+    businessName: "Goodman Grocers",
+    dateRegistered: "2017/04/12",
+    status: "incomplete"
+  },
+  {
+    name: "Freddie Mercury",
+    businessName: "Chocolate Voices",
+    dateRegistered: "2017/03/28",
+    status: "suspended"
+  },
+  {
+    name: "Bill Burr",
+    businessName: "BB's BBQ",
+    dateRegistered: "2020/07/4",
+    status: "closed"
+  },
+]
+
+function UserList (props) {
+  const { data } = props;
```

Comment:
I'm running into an ESLint error when running storybook: "'data' is assigned a value but never used".  Do you have thoughts on how to link this to production as suggested in lines 58 & 59?

Reply:
I was digging around in the master branch, but I'm unsure which component will be calling forth the UserList component or in what state the data will be in once queried from the backend. I would be happy to update the component to reflect the expected design, but would need some direction.

The way the component is written now is that it is expecting an array of each user object. 

 For now I just commented out the data deconstruction on line 47 to remove the error. However, I'm not seeing this error when I run storybook so...

Reply:
Sorry...I realized that ESLint wasn't fully set up in the repo. This was causing me to get errors that nobody else was seeing since I have it enabled. I have an open PR to enable it, but need to have it approved to merge 😂.